bkmr/Cargo.toml

```diff
@@ -40,6 +40,7 @@ tuikit = "0.5.0"
 select = "0.6.0"
 reqwest = {version = "0.11.13", features = ["blocking", "json"] }
 itertools = "0.10.5"
+shellwords = "1.1.0"
```

Comment:
is this dependency really necessary? Looks like it is not actively maintained. What do you think?

Reply:
To me https://crates.io/crates/shellwords is a nice crate, and looks safe and solid.

I tried to remove it, replacing
```rust
let mut fzf_opts_args = shellwords::split(&fzf_opts).unwrap();
```
with
```rust
let mut fzf_opts_args = fzf_opts.split(" ").collect::<Vec<_>>();
```
and it works exactly the same.

So answering your question, shellwords is not necessary.

It might be useful later, in case the value of BKMR_FZF_OPTS should accept more complex options.
But for now, you're right, it's not necessary, so I'll remove it.